apps/examples/security_test/Kconfig

```diff
@@ -0,0 +1,13 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+menu "Security Test"
+source "/root/tizenrt/apps/examples/security_test/hal/Kconfig"
+source "/root/tizenrt/apps/examples/security_test/mbedtls_self/Kconfig"
+source "/root/tizenrt/apps/examples/security_test/seclink/Kconfig"
+source "/root/tizenrt/apps/examples/security_test/security_api/Kconfig"
```

Comment:
Only `security_api` doesn't have `Kconfig_ENTRY`.
Could you check this is okay or not?